_includes/js/queries.md

```diff
@@ -391,3 +424,112 @@ mainQuery.find()
     // There was an error.
   });
 </code></pre>
+
+## Aggregate
+
+Queries can be made using aggregates, allowing you to retrieve objects over a set of input values. The results will not be `Parse.Object`s since you will be aggregating your own fields
+
+* Note: MasterKey is Required. Parse-Server 2.7.1+
+
+Aggregates use stages to filter results by piping results from one stage to the next.
+
+You can create a pipeline using an Array or an Object.
+
+The following example is a pipeline similar to `distinct` grouping by name field.
+
+<pre><code class="javascript">
+// Pipeline object
+const pipeline = {
+  group: { objectId: '$name' }
+ };
+
+// Pipeline array
+const pipeline = [
```

Comment:
Should either rename to 'pipelineArray' or change from **const** to **var** or **let**. In the same example you have two constants declared with the same name, and although it's just an example it should probably show correct code 😉 .